tests/src/test/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBConfigTests.scala

```diff
@@ -114,7 +114,7 @@ class CosmosDBConfigTests extends FlatSpec with Matchers {
       | }
          """.stripMargin).withFallback(globalConfig)
     val cosmos = CosmosDBConfig(config, "WhiskAuth")
-    cosmos should matchPattern { case CosmosDBConfig("http://localhost", "foo", "openwhisk", _, _, _) => }
+    cosmos should matchPattern { case CosmosDBConfig("http://localhost", "foo", "openwhisk", _, _, _, _) => }
```

Comment:
I think some other tests where config is created for each test have the config creation wrapped into a function, so that changing the config sig is a one line change; might be useful here too.

Reply:
Problem here is the need to perform assert on created config. Not able to abstract that out